migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -166,6 +166,23 @@ In {product-title} 3.11, OpenShift SDN was the default networking plugin in Red
 
 For information on migrating to OVN-Kubernetes from OpenShift SDN, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
 
+[WARNING]
+====
```

Comment:
hmm, do we need to mentioned this in this part openshift migration 3 to 4?
what's the meaning add this for migration 3 to 4?

Reply:
My concern is I don't want customers to be surprised by this. That is why I'd like to add this.

Reply:
@zhaozhanqi does the update work for you?

Reply:
Hmm,  I remembered cluster migration from openshift 3.x to 4.x is not supported.  so I'm not sure if this is needed